docs/content/operations/metrics.md

```diff
@@ -33,6 +33,7 @@ Available Metrics
 |`query/node/time`|Milliseconds taken to query individual historical/realtime nodes.|id, status, server.|< 1s|
 |`query/node/bytes`|number of bytes returned from querying individual historical/realtime nodes.|id, status, server.| |
 |`query/node/ttfb`|Time to first byte. Milliseconds elapsed until broker starts receiving the response from individual historical/realtime nodes.|id, status, server.|< 1s|
+|`query/node/backpressure`|Milliseconds channel has spent in an unreadable state.|id, status, server.| |
```

Comment:
Suggested alternate wording that would be clearer:

> Milliseconds that the channel to this node has spent suspended due to backpressure.

Reply:
Thanks for the suggestion! Have reworded the description.